src/javascript/binary/common_functions/form_validation.js

```diff
@@ -242,12 +247,19 @@ const Validation = (() => {
         if (field.$error && field.$error.length) {
             field.$error.setVisibility(0);
         }
+
+        if (field.$error_address && field.$error.length) {
```

Comment:
@binary-hamedanchi should the second condition be `field.$error_address.length`?

Reply:
Noted, will make the change.